lib/middleware/topics/single/topic-get.js

```diff
@@ -3,7 +3,9 @@
 const helpers = require('../../../helpers');
 
 module.exports = function getTopic() {
-  return (req, res) => helpers.topic.getById(req.params.topicId)
+  // TODO: Use helpers.topic.getById once the new broadcast content types have been added as topic
+  // content types. This is a hack to support saving new broadcasts as conversation topics.
+  return (req, res) => helpers.contentfulEntry.getById(req.params.topicId)
```

Comment:
This hack is 🚫 -- this `contenfulEntry` helper is not caching data. Working on a new branch to clean this up (and avoid the hack altogether)

Reply:
Fixed in #1064